src/Machete.HL7.Tests/Segments/BaseHL7Text.cs

```diff
@@ -0,0 +1,104 @@
+﻿namespace Machete.HL7.Tests.Segments
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+
+
+    [Serializable]
+    public abstract class BaseHL7Text :
```

Comment:
Was trying to use as part of a test. How the test harness is currently set up I can only load the default spec or the user-defined spec and I needed some of that core functionality in the user-defined spec